pkg/cloudproviders/aws/certs.go

```diff
@@ -0,0 +1,612 @@
+package aws
+
+import (
+	"crypto/x509"
+
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+// awsCerts lists all known AWS certificates as of time of writing.
+// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/verify-rsa2048.html
+// for the current list of RSA-2048 certificates.
+//
+// Note: THIS CAN BE BRITTLE. THERE IS NO GUARANTEE THESE CERTS WILL LAST FOREVER.
+// See https://github.com/aws/aws-sdk-go/pull/1593#pullrequestreview-70664445.
+// It is probably in everyone's best interest to check on this periodically.
```

Comment:
I have my reservations about this, but I'm not sure about alternatives to it

Reply:
Yeah I don't like it, either, but this seems to be the way Amazon suggests to do it. They just list out a bunch of certificates that you are meant to use to manually check this. This is part of the reason the AWS Go SDK doesn't support this operation, and we need to do it ourselves